stateproof/builder.go

```diff
@@ -252,32 +271,34 @@ func (spw *Worker) getAllOnlineBuilderRounds() ([]basics.Round, error) {
 	return rnds, err
 }
 
-func (spw *Worker) insertSigToBuilder(builderForRound *builder, sig *pendingSig) {
-	rnd := builderForRound.Round
-	pos, ok := builderForRound.AddrToPos[sig.signer]
+var errAddressNotInCommittee = errors.New("cannot find address in builder")              // Address was not a part of Top N accounts in this StateProof's committee
```

Comment:
nits: can you please avoid using the word committee? avoid confusion with algorand's consensus